pkgs/top-level/python-packages.nix

```diff
@@ -26423,4 +26423,75 @@ in modules // {
     };
   };
 
+  sigtools = buildPythonPackage rec {
+    name = "sigtools-${version}";
+    version = "1.1a3";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/sigtools/${name}.tar.gz";
+      sha256 = "190w14vzbiyvxcl9jmyyimpahar5b0bq69v9iv7chi852yi71w6w";
+    };
+
+    # Tests fail with : AttributeError: 'module' object has no attribute 'test_forwards'
+    doCheck = false;
```

Comment:
Eventually, it's possible to nicely handle things with the pypi package and without using nox.
It just needs a little patch to the setup.py, but I proposed the author to port it upstream, so hopefully it won't be needed for long.